README.md

```diff
@@ -781,6 +781,7 @@ This list results from Pull Requests, reviews, ideas, and work done by 1100+ peo
   * [webtranslateit.com](https://webtranslateit.com/) — Free up to 500 strings
   * [weblate.org](https://weblate.org/) — It's free for libre projects with up to 10,000 string sources for the free tier and Unlimited Self-hosted on-premises.
   * [Free PO editor](https://pofile.net/free-po-editor) — Free for everybody
+  * [locize.com](https://locize.com/) — Free for non-profit [open source](https://locize.com/open-source.html) projects
```

Comment:
As per the top of the document this is a list for SaaS services.

Please detail the FREE features of this SaaS?

Reply:
Like described on [this page](https://locize.com/open-source.html), if the open source project fits the requirements it gets a sponsored locize project.

Reply:
Dont really accept services for which you have to go and apply as a OSS project, we might have a few left here but no new ones sorry